conn2res/plotting.py

```diff
@@ -169,41 +179,43 @@ def plot_iodata(
     y = _check_y_dims(y)
 
     # set plotting theme
-    sns.set(style="ticks", font_scale=1.0)
-    fig, ax = plt.subplots(1, 1, figsize=(12, 2))  # 12, 4.5
-
-    # set color palette
-    palette = kwargs.pop('palette', None)
+    rc_defaults = {'figure.titlesize': 12, 'axes.labelsize': 11,
+                   'xtick.labelsize': 11, 'ytick.labelsize': 11,
+                   'legend.fontsize': 8, 'legend.loc': 'best',
+                   'lines.linewidth': 1, 'savefig.format': 'png'}
+    rc_defaults.update(rc_params)
+    sns.set_theme(style='ticks', rc=rc_defaults)
+    
+    # open figure and axes
+    fig_defaults = {'figsize': (12, 2)}  # 12, 4.5
+    fig_defaults.update(fig_params)
```

Comment:
We could change any Figure settings using fig_params, for instance, we could use fig_params={'figsize': (12, 4.5)} if we wanted a figures with diifferent dimensions